client/src/components/common/names-capture.vue

```diff
@@ -673,7 +780,25 @@ export default class NamesCapture extends Vue {
 }
 </script>
 
-<style scoped lang="sass">
-::v-deep .v-messages__message
-  line-height: 14px !important
+<style lang="scss" scoped>
+@import "@/assets/scss/theme.scss";
+
+::v-deep .v-messages__message {
+  line-height: 14px !important;
+}
+
+.main-message-style {
+  font-size: 14px;
+  color: #495057;
+}
```

Comment:
Worth checking the main.scss, this looks like it could be a generic class that already exists.
In the `SASS` file, this is `.copy-small` which extends `.copy-normal`. Might be worth creating some equivalent in our `scss` file, as we are transitioning away from the sass.
Seems fairly common, thoughts here @severinbeauvais ?

Reply:
Maybe `text-body-1` or `text-body-2`.

Reply:
No, those are too large. It's strange since Tracey doesn't usually spec small fonts.

Maybe create a new override style, `text-body-3` or something?